apps/notifications/src/panel/boot/stylesheets/main.scss

```diff
@@ -988,3 +994,58 @@ $with-sidebar-min-page-width: 1114px;
 		}
 	}
 }
+
+.wpnc__keyboard-shortcuts-popover.popover {
+	font: {
+		family: $sans;
+		size: $wpnc__capital-font-size;
+	}
+
+	.popover__arrow {
+		z-index: 1000;
```

Comment:
I was wondering if it make sense to define these z-index values (this and others below) in [_z-index.scss](https://github.com/Automattic/wp-calypso/blob/4715e82025226d84a387a24f1810a25d2fcd75c2/client/assets/stylesheets/shared/functions/_z-index.scss#L4). Although thats really a question and not a suggestion, this is technically an app and that is part of calypso proper. So im not entirely sure if it makes sense or would work as expected in the standalone builds, but that file with z-index definitions came to mind.

Reply:
Good suggestion but not sure what we fallback to outside of calypso in standalone. I'll look into this.